kafka-client/src/test/java/dev/responsive/kafka/internal/db/CassandraKVTableIntegrationTest.java

```diff
@@ -196,6 +197,40 @@ public void shouldReturnRangeKeysInLexicalOrderAcrossMultipleSubPartitions() {
     }
   }
 
+  @Test
```

Comment:
Can we add a test that hits the overflow case? Would be good to have one for each table type to make sure the underlying engine does the right thing